doc/iterations/2025.2/add_loading_indicators_for_long_actions.adoc

```diff
@@ -0,0 +1,56 @@
+= Add loading indicators for long actions
+
+== Problem
+
+Some long actions do not have a loading indicator.
+
+The lack of feedback for the user is an ui/ux issue.
+
+== Key Result
+
+All long actions should provide some kind of loading indicator.
+
+With MUI we can provide 3 types of indicator depending on the situation :
+
+1 - A circular progress bar used to replace a button after a click
+
+2 - A linear progress bar used to replace a component which contains several other component
+
+3 - A skeleton progress that represent a placeholder shape during loading
+
+=== Scenario
+
+==== An user want to open a representation
+
+- Since representations can have many different shapes, we'll show a linear progress bar by default.
+- On table representation, form representation and tree representation we could use a skeleton shape.
+
+==== An user want to execute a long action on a representation
+
+- Afer 1sec, we'll replace the current representation with the same loading indicator as the one used during opening.
```

Comment:
That's a bit much in terms of user interaction. Something more discrete could be to display a small loading indicator (for example a spinner in a corner or something like that) to indicate that we are still waiting for the representation refreshed event payload matching the executed mutation.